apps/services/endorsements/api/infra/endorsement-system-scripts-update-metadata.ts

```diff
@@ -0,0 +1,28 @@
+import { ref, service, ServiceBuilder } from '../../../../../infra/src/dsl/dsl'
+import { settings } from '../../../../../infra/src/dsl/settings'
+
+export const serviceSetup = (services: {
+  servicesTemporaryVoterRegistryApi: ServiceBuilder<'services-temporary-voter-registry-api'>
+}): ServiceBuilder<'endorsement-system-scripts-update-metadata'> =>
+  service('endorsement-system-scripts-update-metadata')
+    .namespace('endorsement-system')
+    .image('services-endorsements-scripts-update-metadata')
+    .serviceAccount()
```

Comment:
you should remove this line, we do not need service account for this one

Reply:
Removed